unlock-app/src/queries/keyholdersByLock.ts

```diff
@@ -58,6 +87,20 @@ const ExpiredKeys = gql`
   }
 `
 
-export default function keyholdersByLockQuery(showActive = true) {
-  return showActive ? ActiveKeys : ExpiredKeys
+const QUERY_BY_TYPE = {
+  [MemberFilters.ACTIVE]: ActiveKeys,
+  [MemberFilters.EXPIRED]: ExpiredKeys,
+  [MemberFilters.ALL]: AllKeys,
+}
```

Comment:
Not a big deal for this but we should deprecate use of enum (they are not recommended to be used by the typescript team)

Reply:
remove the enum with `type` definition